openshift/koku-template.yaml

```diff
@@ -517,14 +519,25 @@ parameters:
   name: SERVICE_ADMIN_USER
   required: true
 - displayName: Service Admin Email
-  from: 'cost-mgmt@redhat.com'
-  generate: expression
+  from: 'cost-mgmt@project-koku.com'
   name: SERVICE_ADMIN_EMAIL
   required: true
 - displayName: Service Admin Password
   from: '[a-zA-Z0-9]{16}'
   generate: expression
   name: SERVICE_ADMIN_PASSWORD
+- displayName: Email Service Host
```

Comment:
I think to pass these params into the container they need to go either in the secret, or configmap. HOST and PORT should probably be in the configmap, IDK if user should be secured in a secret or config map is fine?

Reply:
IMO, usernames are fine in either the configmap or the secret.  

There's a small usability difference in co-locating all of the settings for a given feature into the same object versus splitting them between a secret and a configmap. It's not a huge deal, but it's worth noting. 

There's also the option of having a distinct configmap/secret per-feature, but I think that's only worth exploring if we need to expose pieces of the same data to different pods, like we do with the database creds.

Reply:
🤦‍♂️ I want to blame meetings and task switching for this. Will update.